tic-tac-toe/src/index.css

```diff
@@ -0,0 +1,94 @@
+body {
+    font: 14px "Century Gothic", Futura, sans-serif;
+    margin: 40px;
+    background: #fff;
+}
+
+ol, ul {
+    padding-left: 30px;
+}
+
+.board-row:after {
+    clear: both;
+    content: "";
+    display: table;
+}
+
+.status {
+    color: #424242;
+    margin-bottom: 10px;
+}
+
+.square {
+    color: #424242;
+    background: #9df3f9;
+    border: 4px solid #00897b;
+    float: left;
+    font-size: 96px;
+    font-weight: bold;
+    line-height: 136px;
+    height: 136px;
+    margin-right: -1px;
+    margin-top: -1px;
+    padding: 0;
+    text-align: center;
+    width: 136px;
+}
+
+.square:focus {
+    outline: none;
+}
+
+.kbd-navigation .square:focus {
```

Comment:
Keyboard navigation styles is classy. I am not familiar with this implementation. How does this work in your app?